Gemfile

```diff
@@ -1,11 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf',  '~> 3.2'
-gem 'chefspec',   '~> 4.2'
-gem 'foodcritic', '~> 4.0'
-gem 'rubocop',    '~> 0.30.1'
+gem 'berkshelf', '~> 4.0'
+gem 'chef-zero', '= 4.5'
```

Comment:
Ruby 2.0 is end of life and the Chef omnibus packages ship with 2.1.X so you should just remove 2.0 from testing.